File: swagger/models/update_client_visit_response.py
from .base_model import BaseModel

from .visit import Visit

class UpdateClientVisitResponse(BaseModel):
	"""
	no description available

	Attributes:

		visit
		The updated visit.
	"""

	input_map = {
		'Visit': ('visit', Visit)
		}

	output_map = {
		'visit': 'Visit',
		}

File: swagger/Models/UpdateClientVisitResponse.php
<?php

namespace App\MindBody\Models;

/**
 * no description available
 */
class UpdateClientVisitResponse extends BaseModel
{
	/**
	 * The updated visit.
	 * 
	 * @var Visit
	 */
	public $visit;

	protected function getInputMap()
	{
		return [
			'Visit' => ['visit', Visit::class],
			];
	}

	protected function getOutputMap()
	{
		return [
			'visit' => 'Visit',
			];
	}
}
